src/components/AlertBanner/AlertBanner.tsx

```diff
@@ -27,7 +28,7 @@ interface AlertBannerProps extends VibeComponentProps {
   children?: ReactElement<AlertBannerButtonProps | AlertBannerLinkProps | AlertBannerTextProps>;
 }
 
-const AlertBanner: React.FC<AlertBannerProps> & {
+const AlertBanner: VibeComponent<AlertBannerProps> & {
```

Comment:
just making sure this is intentionally

Reply:
It was, it will correctly pass the ref now